start/routes.ts

```diff
@@ -98,18 +98,20 @@ router
     router.get("/about_us", [AboutUsController, "index"]);
 
     router.get("/newsfeed/latest", [NewsfeedController, "latest"]);
-    router.group(() => {
-      router.get("/cache_reference_number", [
-        CacheReferenceNumberController,
-        "index",
-      ]);
-      router
-        .patch("/cache_reference_number/bump", [
-          CacheReferenceNumberController,
-          "bump",
-        ])
-        .use(middleware.auth());
-    });
+    router
+      .group(() => {
+        router.get("/", [MobileConfigController, "index"]);
+        router
+          .patch("/bump", [MobileConfigController, "bump"])
+          .use(middleware.auth());
```

Comment:
i know this is something the previous PR screwed up, but it appears that it slipped through the review process
using `PATCH` for this kind of endpoint (one that doesn't accept a request body with the edits to make) doesn't make sense. a standard `POST` would be more fitting.